app/models/website.rb

```diff
@@ -48,10 +48,13 @@ def check_release(version)
     return release
   end
 
-  def self.daily_report
-    date = Time.now - 1.day
-    Website.joins(:grouped_issues).where('grouped_issues.updated_at > ?', date).uniq.each do |website|
-      GroupedIssueMailer.notify_daily(website).deliver_later
+  def self.custom_report(date, field)
+    WebsiteMember.where("website_members.#{field} = ?", true).joins(website: :grouped_issues).where('grouped_issues.updated_at > ? AND muted = ?', date, false).uniq.each do |member|
```

Comment:
for these use `... .find_each(batch_size: 500) do |member|` so if we are going to have a gazillion number of users we will get them in batches of 500.